components/RegistrationForm/RegistrationForm.js

```diff
@@ -175,6 +182,20 @@ function RegistrationForm({ initialValues, onSuccess }) {
               disabled={isSubmitting}
               autoComplete="postal-code"
             />
+
+            <Field
+              name="codeOfConduct"
+              label="I have read the Code of Conduct"
+              component={Checkbox}
+              disabled={isSubmitting}
+            />
+
+            <Field
+              name="communityGuidelines"
+              label="I have read the Community Guideliens"
```

Comment:
Assuming you allowed for node type via prop-types...

```suggestion
              label={
                <>
                  I have read the{' '}
                  <OutboundLink
                    hasIcon={false}
                    href="some-url.com"
                    analyticsEventLabel="Looked at Code of Conduct"
                  >
                    Code of Conduct
                  </OutboundLink>
                </>
              }
```